egs/librispeech/asr/simple_v1/run.sh

```diff
@@ -3,11 +3,36 @@
 # Copyright 2020 Xiaomi Corporation (Author: Junbo Zhang)
 # Apache 2.0
 
-# Example of how to build L and G FST for K2. Most scripts of this example are copied from Kaldi.
-
 set -eou pipefail
 
-stage=0
+libri_dirs=(
+/root/fangjun/data/librispeech/LibriSpeech
+/export/corpora5/LibriSpeech
+/home/storage04/zhuangweiji/data/open-source-data/librispeech/LibriSpeech
+/export/common/data/corpora/ASR/openslr/SLR12/LibriSpeech
+)
+
+libri_dir=
+for d in ${libri_dirs[@]}; do
+  if [ -d $d ]; then
+    libri_dir=$d
+    break
+  fi
+done
+
+if [ ! -d $libri_dir/train-clean-100 ]; then
+  echo "Please set LibriSpeech dataset path before running this script"
+  exit 1
+fi
+
+echo "LibriSpeech dataset dir: $libri_dir"
+
+stage=5
+
+# settings for BPE training -- start
+vocab_size=5000
```

Comment:
@danpovey 

I've removed the bigram P and it is possible to do BPE LF-MMI training with vocab_size=5000.